tests/unit/test-cmd/test.lint.js

```diff
@@ -66,6 +67,14 @@ describe('lint', () => {
     });
   });
 
+  it('allow deactivate runAsBinary param into linter', () => {
```

Comment:
I'm not sure about this. I think maybe could be `it ... set runAsBinary according shouldExitProgram option` ?